src/App.js

```diff
@@ -1,21 +1,17 @@
 import React, { Component } from "react";
-import logo from "./logo.svg";
 import "./App.scss";
-import {BrowserRouter, Route, Switch, NavLink } from "react-router-dom";
+import {BrowserRouter, Route, Switch } from "react-router-dom";
 import "./App.sass";
-import Dummy from "./js/components/Dummy";
 import Home from "./js/components/Home/Home";
-import Navbar from "./js/components/Navbar/Navbar";
-import { Provider } from "react-redux";
 import Books from "./js/components/Books/Books";
 import Login from "./js/components/LoginPage/Login";
 import SignUp from "./js/components/RegisterPage/SignUp";
 import ForgotSignIn from "./js/components/ForgotSignIn/ForgotSignIn";
 
 class App extends Component {
   render() {
-    var loc_navBarTitle = "KnowYourNation";
-    var loc_navbarItems = [false, true, true, true];
+    // var loc_navBarTitle = "KnowYourNation";
+    // var loc_navbarItems = [false, true, true, true];
```

Comment:
Well, if you paid attention to what raised the fierce outrage of your contributors, you could notice that Ghost codes were on top of the list. By ghost code it simply means codes that are of no use in your application.
Remove them.